bigtable/client/filters_test.cc

```diff
@@ -0,0 +1,94 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/filters.h"
+
+#include <gmock/gmock.h>
+
+#include "bigtable/client/chrono_literals.h"
+
+namespace btproto = ::google::bigtable::v2;
+
+TEST(FiltersTest, PassAllFilter) {
+  auto proto = bigtable::Filter::PassAllFilter().as_proto();
+  EXPECT_TRUE(proto.pass_all_filter());
+}
+
+TEST(FiltersTest, BlockAllFilter) {
+  auto proto = bigtable::Filter::BlockAllFilter().as_proto();
+  EXPECT_TRUE(proto.block_all_filter());
+}
+
+TEST(FiltersTest, Latest) {
+  auto proto = bigtable::Filter::Latest(3).as_proto();
+  EXPECT_EQ(3, proto.cells_per_column_limit_filter());
+}
+
+TEST(FiltersTest, ColumnRegex) {
+  auto proto = bigtable::Filter::ColumnRegex("col[A-E]").as_proto();
+  EXPECT_EQ("col[A-E]", proto.column_qualifier_regex_filter());
+}
+
+TEST(FiltersTest, ColumnRange) {
+  auto proto = bigtable::Filter::ColumnRange("colA", "colF").as_proto();
+  ASSERT_EQ(btproto::ColumnRange::kStartQualifierClosed,
```

Comment:
Why can't all checks here be `EXPECT` in this test case? Failing them will not crash the test.

Reply:
Yup.  Fixed.